bazel/repositories.bzl

```diff
@@ -172,11 +172,11 @@ def proxy_wasm_cpp_host_repositories():
 
     maybe(
         http_archive,
-        name = "llvm-13_0_1",
+        name = "llvm-prj-15_0_7",
```

Comment:
Nit: Please either leave it as `llvm-15_0_7` or change to `llvm-project-15_0_7`. `prj` is an unnecessary shortening.